### PR TITLE
Feature: Use existing frames

### PR DESCRIPTION
**Pull request by kalisp**
## Changelog Description
This PR adds option to publish previously locally rendered files. This allows artist to prepare workfile, press AE `Render` button and visually check rendered files. Only after that they might decide that these files are OK and should be published without re-rendering.

## Additional info
Similar to Nuke functionality.
This requires reinstall of extension!!!

## Testing notes:

1. set `Use existing frames` in Creator
image
3. check Validator function before you'll hit Render in AE
4. Hit `Render`
image
5.Publish

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
Maybe the `RenderCreator` should convert value of `"farm"` from older instances to render target.

File: client/ayon_aftereffects/plugins/create/create_render.py [hunk 1]
```diff
@@ -257,4 +274,13 @@ def _handle_legacy(self, instance_data):
         if instance_data["creator_attributes"].get("mark_for_review") is None:
             instance_data["creator_attributes"]["mark_for_review"] = True
 
+        farm = instance_data["creator_attributes"].get("farm")
```
Comment (iLLiCiTiT) [inline client/ayon_aftereffects/plugins/create/create_render.py:277]:
```suggestion
        farm = instance_data["creator_attributes"].pop("farm")
```

File: client/ayon_aftereffects/plugins/create/create_render.py [hunk 2]
```diff
@@ -257,4 +274,13 @@ def _handle_legacy(self, instance_data):
...
+        if farm is not None:
+            if farm:
+                instance_data["creator_attributes"]["render_target"] = "farm"
+            else:
+                instance_data["creator_attributes"]["render_target"] = "local"
+
+            instance_data["creator_attributes"].pop("farm")
```
Comment (iLLiCiTiT) [inline client/ayon_aftereffects/plugins/create/create_render.py:284]:
```suggestion
```

**Review by 64qam — APPROVED** [review]

File: client/ayon_aftereffects/plugins/create/create_render.py (hunk 1 above)
Comment (kalisp) [reply client/ayon_aftereffects/plugins/create/create_render.py]: Ok

**Pull request merged**